src/components/LoginButton.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+
+const LoginButton = (): JSX.Element => {
+  const [loggedin, setLoggedIn] = React.useState<Boolean>(false);
+  const [text, setText] = React.useState<String>("");
+
+  const fetchlogin = () => {
+    fetch("/id", {
+      method: "GET",
+    })
+      .then(function (res) {
+        return res.text();
+      })
+      .then(function (id) {
+        if (id !== "") {
+          setLoggedIn(true);
+          setText("Logout");
+        } else {
+          setLoggedIn(false);
+          setText("Log In With Github");
+        }
+      });
+    console.log("fetching login");
+  };
+
+  const handleUser = () => {
+    if (loggedin) {
+      logout();
+    } else {
+      userlogin();
+    }
+  };
+
+  const logout = () => {
+    fetch("http://localhost:3000/logout", {
```

Comment:
I'm experiencing an error with logout functionality where the login button won't update upon clicking logout (it _does_ log the user out, but doesn't redirect their page accordingly.) Attached is a video the unwanted behavior:

https://user-images.githubusercontent.com/51765298/136437290-e33eded4-8cb7-432f-a107-0b34799b4b2a.mp4

- [ ] Fix Logout Button Redirect

Reply:
I'm unable to recreate your error atm.  Could you try pulling latest changes and see?

Reply:
Actually I changed my mind again... I stopped the redirect because this app is single page and we really don't need it I just send a 200 and message saying successful should be good.  Try again and this should be golden